e2e/suite.go

```diff
@@ -90,6 +91,7 @@ func Run(t *testing.T) {
 	defer os.Remove(imagePath)
 
 	// RunE2ETests by functionality
-	t.Run("BUILD", imgbuild.RunE2ETests)
-	t.Run("ACTIONS", actions.RunE2ETests)
+//	t.Run("BUILD", imgbuild.RunE2ETests)
+//	t.Run("ACTIONS", actions.RunE2ETests)
```

Comment:
un-comment

Reply:
Yeah, I comment that so I can test faster, still a WIP... :+1: